vendor/engines/secure_rooms/app/controllers/secure_rooms_api/scans_controller.rb

```diff
@@ -14,4 +16,10 @@ def scan
     render json: response_json, status: :forbidden
   end
 
+  def load_models
+    @user = User.find_by!(card_number: params[:card_id])
+    @control_device = ControlDevice.find(params[:controller_id])
+    @card_reader = CardReader.find(params[:reader_id])
```

Comment:
We might also need to add a custom `rescue_from` to these controllers. We don't want to send back HTML if we can avoid it. This might be a question for Dan, but should we return in the body why it's 404ing?